torch/testing/_internal/common_methods_invocations.py

```diff
@@ -6898,6 +6907,12 @@ def reference_logsigmoid(x):
         -np.log1p(np.exp(-x)))
 
 
+def reference_hardsigmoid(x):
+    int = x / 6 + 0.5
+    y = np.where(int < 0, 0, int)
```

Comment:
nit: don't use "int" as a name because it's a Python builtin (https://docs.python.org/3/library/functions.html#int)

Instead of calling `np.where` twice I think this can just call `np.clip`? (https://numpy.org/doc/stable/reference/generated/numpy.clip.html?highlight=clip#numpy.clip)